src/main/java/com/iota/iri/service/API.java

```diff
@@ -1027,49 +1378,96 @@ private AbstractResponse getBalancesStatement(final List<String> addresses, fina
             instance.milestoneTracker.latestSnapshot.rwlock.readLock().unlock();
         }
 
-        final List<String> elements = addressList.stream().map(address -> balances.get(address).toString())
+        final List<String> elements = addressList.stream()
+                .map(address -> balances.get(address).toString())
                 .collect(Collectors.toCollection(LinkedList::new));
 
-        return GetBalancesResponse.create(elements, hashes.stream().map(h -> h.toString()).collect(Collectors.toList()), index);
+        return GetBalancesResponse.create(elements, hashes.stream()
+                .map(h -> h.toString())
+                .collect(Collectors.toList()), index);
     }
 
     private static int counter_PoW = 0;
+    
+    /**
+     * Can be 0 or more, and is set to 0 every 100 requests.
+     * Each increase indicates another 2 tips send.
+     * 
+     * @return The current amount of times this node has done proof of work.
+     */
     public static int getCounterPoW() {
         return counter_PoW;
     }
+    
+    /**
+     * Increases the amount of times this node has done proof of work by one.
+     */
     public static void incCounterPoW() {
         API.counter_PoW++;
     }
 
     private static long ellapsedTime_PoW = 0L;
+    
+    /**
+     * Can be 0 or more, and is set to 0 every 100 requests.
+     * 
+     * @return The current amount of time spent on doing proof of work in milliseconds.
+     */
     public static long getEllapsedTimePoW() {
         return ellapsedTime_PoW;
     }
+    
+    /**
+     * Increases the current amount of time spent on doing proof of work.
+     * 
+     * @param ellapsedTime the time to add, in milliseconds.
+     */
     public static void incEllapsedTimePoW(long ellapsedTime) {
         ellapsedTime_PoW += ellapsedTime;
     }
 
     /**
-      * Attaches the specified transactions (trytes) to the Tangle by doing Proof of Work.
-      * You need to supply <code>branchTransaction</code> as well as <code>trunkTransaction</code> (the tips which you're going to validate and reference with this transaction) - both of which you'll get through the <code>getTransactionsToApprove</code> API call.
-      *
-      * The returned value is a different set of tryte values which you can input into <code>broadcastTransactions</code> and <code>storeTransactions</code>.
-      * The last 243 trytes of the return value consist of the: <code>trunkTransaction</code> + <code>branchTransaction</code> + <code>nonce</code>.
-      * These are valid trytes which are then accepted by the network.
-      *
-      * @param trunkTransaction the trunk transaction
-      * @param branchTransaction the branch transaction
-      * @param minWeightMagnitude the minimum weight magnitute
-      * @param trytes the list of trytes to attach
-      * @return trytes the list of transactions in trytes
+      * <p>
+      *     Attaches the specified transactions (trytes) to the Tangle by doing Proof of Work.
```

Comment:
this is always confusing as it doesn't actually attach to the tangle, would be good to mention that, and the need to do store/broadcast.